fix(chip): accessibility focus issue on chip

realdreamer opened the pull request:
Reviewers: @MLKiiwy @kagnamazian @realdreamer

Preview: https://static-staging.usabilla.com/stylabilla/pr/fix-chip-accessibility/docs/section-components.html

realdreamer commented:
@MLKiiwy as you mentioned removed browsers default outline for focus. 

@kagnamazian I'll check with @galagmon for button focus and will update based on that

kagnamazian commented:
@realdreamer I think we should keep as much as possible the default outline for focus. But let's see with the design team. She is aware of accessibility. I trust her on every choice she decides.

realdreamer commented:
@kagnamazian checked with @galagmon she's fine with button focus less visibility and removed default browser outline and changed error-state focus border color

ub-jenkins commented:
:tada: This PR is included in version 18.3.1 :tada:

The release is available on:
- [npm package (@latest dist-tag)](https://www.npmjs.com/package/@usabilla/stylabilla)
- [GitHub release](https://github.com/usabilla/stylabilla/releases/tag/v18.3.1)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
